app/javascript/packs/toolbar-actions.js

```diff
@@ -0,0 +1,66 @@
+import { subscribeToRx, DELETE_EVENT } from '../helpers/rxConnector';
+
+const API = angular.injector(['ng', 'miq.api']).get("API");
+
+export function showMessage(messages, success) {
+  if (typeof messages === 'string') {
+    add_flash(messages, success ? 'success' : 'error');
+  } else {
+    Object.keys(messages).forEach((msgStatus) => {
+      const statusKey = msgStatus === 'true';
+      if (statusKey && messages[statusKey] > 0) {
+        add_flash(__(`Deleting of ${messages[statusKey]} items queued.`), 'success');
```

Comment:
is this string properly internationalized?

Reply:
It's not, we need sprintf :)